### DISCUSSION TITLE
Why does sodium/version.h.in exist but not version.h?

### DISCUSSION THREAD

**Discussion by matkatmusic** [jedisct1/libsodium#1421] [Q&A] [answered]
I'm including LibSodium in a desktop app project and i continuously get an error about `sodium/version.h not found`.
I have compiled/installed sodium from source, but when including the headers in my project, this error pops up because `version.h.in` exists. 
I end up renaming that file to `version.h` whenever I load the project up onto a new platform for testing, which is a pretty tedious process and one less thing I'd like to have to remember. 

Am I doing something wrong when integrating Libsodium into my project?
Is there some automation step I'm forgetting to do when building libsodium from source that generates this header file?

```
`version.h` is automatically created by the `configure` script.
```
_Originally posted by @jedisct1 in https://github.com/jedisct1/libsodium/issues/213#issuecomment-65177000_

**Comment by jedisct1** [comment, marked as answer]
`version.h.in` is only used internally by the build process. It's not part of the files that are installed.

If you are using autoconf, when you type `make install`, the library files are installed in `${prefix}/lib` and the header files in `${prefix}/include`. With `prefix` being `/usr/local` by default, or whatever follows the `--prefix` parameter given to the `./configure` command. This is what you should use in your applications.

If you are using Zig, after you type `zig build -Doptimize=ReleaseFast`  (or whatever), the libraries are in `zig-out/lib` and the header files in `zig-out/include`. Once again, these are the files you should use in your application.

But your operating system probably has packages for libsodium already, so you can simply install them, and everything will be in your default compilation paths.

**Reply by matkatmusic** [reply]
Ah, ok.  I was using the `/include` folder in the repository!
thanks for clarifying!!
